schema/schema.json

```diff
@@ -0,0 +1,288 @@
+          "clearKeychain": { "type": "boolean" },
+          "stopApp": { "type": "boolean" }
+        }
+      },
+      "pressKey": {
+        "title": "Press a set of special keys",
+        "type": "string",
+        "enum": [
+          "home",
+          "lock",
+          "enter",
+          "backspace",
+          "volume up",
+          "volume down",
+          "back",
+          "Remote Dpad Up",
+          "Remote Dpad Down",
+          "Remote Dpad Left",
+          "Remote Dpad Right",
+          "Remote Dpad Center",
+          "Remote Media Play Pause",
+          "Remote Media Stop",
+          "Remote Media Next",
+          "Remote Media Previous",
+          "Remote Media Rewind",
+          "Remote Media Fast Forward"
+        ]
+      },
+      "repeat": {
+        "title": "Repeat a set of commands.",
+        "type": "object",
+        "additionalProperties": false,
+        "properties": {
+          "times": {
+            "title": "Repeat a set of commands N times.",
+            "type": "integer"
+          },
+          "while": {
+            "title": "Repeat a set of commands until a condition satisfies.",
+            "$ref": "#/$defs/Condition"
+          },
+          "commands": {
+            "type": "array",
+            "items": {
+              "$ref": "#/items"
+            }
+          }
+        },
+        "required": ["commands", "times"]
+      },
+      "runFlow": {
+        "title": "Runs a flow from a specified file.",
+        "type": "object",
+        "additionalProperties": false,
+        "properties": {
+          "file": {
+            "type": "string"
+          },
+          "env": {
+            "type": "object"
+          },
+          "when": {
+            "type": "object",
+            "additionalProperties": true,
+            "properties": {
+              "visible": { "$ref": "#/$defs/Element" },
+              "notVisible": { "$ref": "#/$defs/Element" }
+            }
+          }
+        }
+      },
+      "runScript": {
+        "title": "Run a provided JavaScript file.",
+        "properties": {
+          "file": {
+            "type": "string"
+          },
+          "env": {
+            "type": "object"
+          }
+        }
+      },
+      "scrollUntilVisible": {
+        "title": "Scroll towards a direction until an element becomes visible in the view hierarchy.",
+        "type": "object",
+        "additionalProperties": false,
+        "properties": {
+          "element": {
+            "$ref": "#/$defs/Element"
+          },
+          "direction": {
+            "$ref": "#/$defs/Direction"
+          },
+          "timeout": { "type": "integer" },
+          "speed": { "type": "integer" },
+          "visibilityPercentage": { "type": "integer" }
+        }
+      },
+      "swipe": {
+        "title": "Perform a swipe gesture.",
+        "type": "object",
+        "additionalProperties": false,
+        "properties": {
```

Comment:
There are a few other ways how swipes can be done:
https://maestro.mobile.dev/api-reference/commands/swipe

`"start":{
              "type": "string"
            },
            "end": {
              "type": "string"
            }`
            
         I don't know JSON schema that well but that works for me